cmake/compilers.cmake

```diff
@@ -20,7 +20,7 @@ function(CHECK_COMPILERS)
         endif()
```

Comment:
Can you also document the procedure of building with GCC through readme?

Reply:
Done